src/test/scripts/functions/builtin/mice.dml

```diff
@@ -19,26 +19,44 @@
 #
 #-------------------------------------------------------------
 
-X = read($X, data_type="frame", format="csv");
```

Comment:
Looking at this test, it seems very complicated to use MICE.
does it make sense to split this up into multiple smaller tests?
(does not have to be in this PR)

Reply:
I think some comments will help to understand.